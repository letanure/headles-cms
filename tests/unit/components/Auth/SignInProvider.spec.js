import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Auth/SignInProvider.vue'

describe(`${Component.name}.vue`, () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      propsData: {
        provider: 'facebook',
      },
      mocks: {
        $t: (msg) => msg,
      },
    })
  })

  describe('smoke tests', () => {
    it('should render', () => expect(wrapper.exists()).toBe(true))
    it('matches snapshot', () => expect(wrapper.element).toMatchSnapshot())
  })
})
