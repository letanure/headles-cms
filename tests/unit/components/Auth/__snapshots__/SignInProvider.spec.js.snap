// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInProvider.vue smoke tests matches snapshot 1`] = `
<el-button-stub
  class="SignInProvider"
  icon=""
  nativetype="button"
  type="default"
>
  SignInProvider.facebook.label
</el-button-stub>
`;
